radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt

```diff
@@ -221,11 +244,15 @@ class RadialGraph : ConstraintLayout {
 
                 val sectionNormalizedSize: BigDecimal = section.normalizedValue
 
-                val labelPositionValue: Float = calculateLabelPositionValue(section, labelStartPositionValue)
+                val labelPositionValue: Float =
+                    calculateLabelPositionValue(section, labelStartPositionValue)
```

Comment:
Not my repo, but the column width seems fairly low?

Reply:
@DuartBreedt should this be adjusted ?

Reply:
Please 🤗